pages/docs/couple-your-code/couple-your-code-direct-access.md

```diff
@@ -12,9 +12,10 @@ This concept is required if you want to access received meshes directly. It migh
 ```cpp
     // Allocate a bounding-box vector containing lower and upper bounds per
     // space dimension
-    std::vector<double> boundingBoxes(dim * 2);
+    std::vector<double> boundingBox(dim * 2);
```

Comment:
Seems that this commit does not belong here.

Reply:
Yup